src/utils/selectors.ts

```diff
@@ -114,6 +114,14 @@ export const shouldRetainAttendanceSelector: SparkleSelector<boolean> = (
   state
 ) => state.attendance.retainAttendance;
 
+export const isCurrentVenueNGRequestedSelector: SparkleSelector<boolean> = makeIsRequestedSelector(
+  "currentVenueNG"
+);
+
+export const isCurrentVenueNGRequestingSelector: SparkleSelector<boolean> = makeIsRequestingSelector(
+  "currentVenueNG"
+);
```

Comment:
It would probably make sense to include return these from the `useConnectCurrentVenueNG` hook, rather than needing to use them individually/separately external to it.